WordPress/Classes/ViewRelated/NUX/SigninHelpers.swift

```diff
@@ -6,10 +6,14 @@ import WordPressShared
 /// A collection of helper methods for NUX.
 ///
 @objc class SigninHelpers: NSObject {
-    fileprivate static let AuthenticationEmailKey = "AuthenticationEmailKey"
     fileprivate static let WPComSuffix = ".wordpress.com"
     @objc static let WPSigninDidFinishNotification = "WPSigninDidFinishNotification"
 
+    fileprivate enum Constants {
+        static let AuthenticationInfoKey = "AuthenticationInfoKey"
```

Comment:
Nipticky: Should this one be lower case as well? (can't recall if we actually have a guideline for this!)

Reply:
We do.  It should.  I will make it so. :)